NeutronStandard/Sniffs/Functions/TypeHintSniff.php

```diff
@@ -12,12 +12,12 @@ public function register() {
 	}
 
 	public function process(File $phpcsFile, $stackPtr) {
-		$this->helper = new SniffHelpers();
-		$this->checkForMissingArgumentHints($phpcsFile, $stackPtr);
-		$this->checkForMissingReturnHints($phpcsFile, $stackPtr);
+		$helper = new SniffHelpers();
+		$this->checkForMissingArgumentHints($phpcsFile, $stackPtr, $helper);
+		$this->checkForMissingReturnHints($phpcsFile, $stackPtr, $helper);
 	}
 
-	private function checkForMissingArgumentHints(File $phpcsFile, $stackPtr) {
+	private function checkForMissingArgumentHints(File $phpcsFile, $stackPtr, SniffHelpers $helper) {
```

Comment:
This is a great idea. I should have done this rather than keeping `helper` as state. Thanks!

Reply:
Did it because you wer using a non-defined helper property which was marked as error in my IDE :)